data_validation/cli_tools.py

```diff
@@ -168,9 +172,34 @@ def configure_arg_parser():
     _configure_find_tables(subparsers)
     _configure_raw_query(subparsers)
     _configure_beta_parser(subparsers)
+    _configure_partition_parser(subparsers)
     return parser
 
 
+def _configure_partition_parser(subparsers):
+    """Configure arguments to generate partitioned config files."""
+    partition_parser = subparsers.add_parser(
+        "generate-partitions",
```

Comment:
Maybe we can change this to `--generate-table-partitions` so it's easy to understand what it's partitioning and how it's different from random row

Reply:
Yes. Done